server/repository/src/migrations/v2_07_00/add_patient_link_id_to_vaccination.rs

```diff
@@ -32,9 +42,10 @@ impl MigrationFragment for Migrate {
                     WHERE program_enrolment.id = vaccination.program_enrolment_id
                 );
 
-                INSERT INTO changelog (table_name, record_id, row_action, store_id)
-                    SELECT 'vaccination', id, 'UPSERT', store_id
-                    FROM vaccination;
+                INSERT INTO changelog (table_name, record_id, row_action, store_id, name_link_id)
+                    SELECT 'vaccination', id, 'UPSERT', store_id, patient_link_id
+                    FROM vaccination
+                    WHERE patient_link_id <> '';
```

Comment:
Hi @andreievg - lol sorry we've gone through this a few times, and I know low likelihood of vaccination records existing. But not inserting name_link_id with changelog here meant that if had existing vaccinations were created on OMS Central, they'd never end up with a changelog with name_link_id populated (that was only happening after remote site push). 

These would then not sync out if patient visibility added. Best to insert changelog correctly with name_link_id, only where name_link_id does exist. Am I missing an edge case where this could error?